tensorflow/compiler/xla/service/gpu/ir_emitter_unnested.cc

```diff
@@ -1497,7 +1497,7 @@ std::unique_ptr<KernelThunk> IrEmitterUnnested::BuildKernelThunk(
     llvm::Type* int8_double_pointer =
         llvm::PointerType::get(b_.getInt8PtrTy(), /*AddressSpace=*/0);
     for (int64 idx : gte_index) {
-      loc = BitCast(loc, int8_double_pointer);
+      loc = PointerBitCastOrAddrSpaceCast(loc, int8_double_pointer);
```

Comment:
Optional, but I would prefer keeping the `builder.Create...` rather than creating a mixin wrapper, as the latter breaks goto-definition and argument-lookup IDE functionality.

Reply:
Agreed on the issue mixin wrapper brings, how should I judge when something qualifies for an interface change in mixin wrapper?